docs/source/tutorial/documentation.rst

```diff
@@ -313,7 +313,7 @@ and follow some simple setup procedures. Not only will it host your Sphinx docum
 Type Annotations
 ^^^^^^^^^^^^^^^^
 
-A relatively new concept in Python,
+A relatively new concept in Python (3.5+),
```

Comment:
Indeed. Might also be worth noting that this is still a growing feature set. Forward references on things like class methods that return an instance of `self` don't work until 3.7+.